### DISCUSSION TITLE
CartesianChart.MouseMove not firing on .NET6/WinForms

### DISCUSSION THREAD

**Discussion by RodLeyton** [Live-Charts/LiveCharts2#397] [Q&A]
Win11/WinForms on .NET6.0 using C#10 on VS2022.
I am trying to receive mouse move events, so I can get the X and Y values via CartesianChart.ScaleUIPoint.

I have registered for the event using 
```
CartesianChart.MouseMove += Chart_MouseMove_new;


Chart_MouseMove_new(object? sender, MouseEventArgs? e) 
{   // Breakpoint
   if (e is null) return;
   //........
}
```

However it does not fire (tested by debug breakpoint).
Also tried MouseClick and MouseDown events, which I couldn't get to fire either.
I could get ChartPointPointerDown to fire successfully, but I really do need the move event.

Are these events supported yet, or am I missing something here?

**Comment by En-JON-eer** [comment]
Bump. 

The MouseDown and MouseUp events fire just fine. But MouseMove does not.

![wfaypV1uql](https://github.com/beto-rodriguez/LiveCharts2/assets/89985196/42fc9ca9-e999-4781-a2e4-ea5b2976995d)

Trying to get Scrollable Charts working with Beta.920 same issue on Beta.900

https://livecharts.dev/docs/WinForms/2.0.0-beta.920/samples.general.scrollable#form-code-behind

**Comment by beto-rodriguez** [comment]
I was able to reproduce the issue, I transferred this to #1209.
